tests/engine/components/Camera.test.cpp

```
@@ -26,14 +26,14 @@ class DummyFramebuffer : public nexo::renderer::Framebuffer {
 public:
     void bind() override {}
     void unbind() override {}
-    void setClearColor(const glm::vec4 &color) override {}
+    void setClearColor(const glm::vec4 &) override {}
```

Cppcheck (2.10-2)

[style] The function 'setClearColor' is never used.

(unusedFunction)